### DISCUSSION TITLE
求助，关于Win安装包问题

### DISCUSSION THREAD

**Discussion by oongvx** [tw93/Pake#520] [Q&A]
1、请问一下在哪里能够修改Win安装包的默认安装路径呢？
2、请问一下在哪里设置，安装完成后默认勾选这个Launch呢？万分感谢！
![1](https://github.com/tw93/Pake/assets/138094961/9107f4c7-6998-479e-8d04-58896336c939)

**Comment by Tlntin** [comment]
能看懂这个就可以改：
https://github.com/tw93/Pake/blob/master/src-tauri/assets/main.wxs

wxs这个语法很变态，我也是copy过来的，你可以试试魔改一下。
